<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenegotiateContractWindow</class>
 <widget class="QWidget" name="RenegotiateContractWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="9" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="LastName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 500 11pt &quot;Montserrat Medium&quot;;
}
</string>
       </property>
       <property name="text">
        <string>Информация о клиенте:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="Patronymic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="3">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1">
      <widget class="PhoneEdit" name="Phone">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
       </property>
       <property name="maxLength">
        <number>18</number>
       </property>
       <property name="placeholderText">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="BackButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 500 14pt &quot;Montserrat Medium&quot;;
    background-color: #c8c8c8; 
    color: white; 
    border: none;
    font-size: 14px;
    border-radius: 5px; 
}
QPushButton:hover {
    background-color: #aeaeae;
}
QPushButton:pressed {
    background-color: #949494; 
}</string>
         </property>
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 500 11pt &quot;Montserrat Medium&quot;;
}
</string>
       </property>
       <property name="text">
        <string>Информация о договоре:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="FirstName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="SendButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 14pt &quot;Montserrat Medium&quot;;
    background-color: #4CAF50; 
    color: white; 
    border: none;
    font-size: 14px;
    border-radius: 5px; 
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed {
    background-color: #357a38; 
}</string>
         </property>
         <property name="text">
          <string>Отправить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 500 11pt &quot;Montserrat Medium&quot;;
}
</string>
       </property>
       <property name="text">
        <string>Комментарий бухгалтера:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QTextEdit" name="AccountantComments">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
	font: 500 11pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
	selection-color: white;
}

QTextEdit:focus {
    border-color: #5fbaeb;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Комментарий</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="3">
      <widget class="NumberEdit" name="Summa">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Сумма</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QComboBox" name="TypeContract">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QComboBox:editable {
    background: #5fbaeb;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: #cdcdcd;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px;
	border-left-width: 1px;
}

QComboBox::down-arrow {
   image: url(:/images/resources/CheckMark.png);
	width: 12px;
	height: 12px;
}
QComboBox QAbstractItemView::item {
height: 40px;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Выберите тип договора</string>
       </property>
       <item>
        <property name="text">
         <string>Добровольное медицинское страхование</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cтрахование домашнего имущества</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Страхование автотранспорта</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhoneEdit</class>
   <extends>QLineEdit</extends>
   <header>phoneedit.h</header>
  </customwidget>
  <customwidget>
   <class>NumberEdit</class>
   <extends>QLineEdit</extends>
   <header>numberedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LastName</tabstop>
  <tabstop>FirstName</tabstop>
  <tabstop>Patronymic</tabstop>
  <tabstop>Phone</tabstop>
  <tabstop>TypeContract</tabstop>
  <tabstop>Summa</tabstop>
  <tabstop>BackButton</tabstop>
  <tabstop>SendButton</tabstop>
  <tabstop>AccountantComments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
